setup.py

```diff
@@ -0,0 +1,88 @@
+# To use a consistent encoding
+from codecs import open
+from os import path
+from setuptools import setup, find_packages, Command
+
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as readme:
+    long_description = readme.read()
+
+lambda_name = 'ami-cleaner'
+
+# Versions should comply with PEP440.  For a discussion on single-sourcing
+# the version across setup.py and the project code, see
+# https://packaging.python.org/en/latest/single_source_version.html
+lambda_version = '0.0.1'
+
+lambda_description = 'Lambda function that can be scheduled to clean up Old AMIs'
+
+# How mature is this project? Common values are
+#   3 - Alpha
+#   4 - Beta
+#   5 - Production/Stable
+lambda_dev_status = '4 - Beta'
+
+# What does your project relate to?
+lambda_keywords = 'AWS Lambda CodeBuild'
+
+# What is your project's license
+lambda_license = 'APL 2.0'
+
+# Who is the author?
+lambda_author = 'Vinay Kumar'
+lambda_author_email = 'vchinnakotla@quinovas.com'
+
+# List run-time dependencies here.  These will be installed by pip when
+# your project is installed. For an analysis of "install_requires" vs pip's
+# requirements files see:
+# https://packaging.python.org/en/latest/requirements.html
+lambda_install_requires = []
+
+##############################################
+# CHANGES BELOW HERE ARE MADE AT YOUR OWN RISK
+##############################################
+
+setup(
+    name=lambda_name,
+
+    version=lambda_version,
+
+    description=lambda_description,
+    long_description=long_description,
```

Comment:
long_description needs to be defined, or you need to change this to point to lambda_description